scripts/generate_env.py

```diff
@@ -0,0 +1,39 @@
+## This script is included in our .devcontainers setup.  We basically use this to generate our an environment variables file that can be used to set the Genesys Cloud environment variables.
```

Comment:
Simple python script I am including in the containers to help set environment variables